<launch>
  <!-- 启动 kdl_solver 节点 -->
  <node name="payload_estimator_node" pkg="payload_estimator" type="payload_estimator" output="screen" launch-prefix="gdb -ex run --args" />
</launch>